server/src/main/java/org/opensearch/action/admin/cluster/state/term/GetTermVersionAction.java

```diff
@@ -18,7 +18,7 @@
 public class GetTermVersionAction extends ActionType<GetTermVersionResponse> {
 
     public static final GetTermVersionAction INSTANCE = new GetTermVersionAction();
-    public static final String NAME = "cluster:monitor/term";
+    public static final String NAME = "internal:monitor/term";
```

Comment:
@peternied I understood the issue, but it looks like this is a cluster action and has to start with `cluster:` (like all cluster actions), could we add permissions to the security plugin instead?

Reply:
> has to start with `cluster:`

@reta Why does it have to be the case? The feature is a freshness check before fetching the cluster state, seems pretty internal to me.  Looking around I found no clear convention such as namespacing that would clarify, seems relatively adhoc.


Side note, in the security plugin we have a special exception for any tasks starting with `internal:`, if we updated it, it would break that convention for this new task, we'd need to make change in many places in the security plugin to ensure its accommodated.  I suspect other security implementations would have the same trouble.

Reply:
> @reta Why does it have to be the case?

Every single action under `admin/cluster` is cluster action, so I have doubts just changing the name is the solution - the action may be moved out of cluster actions altogether